### DISCUSSION TITLE
Feature request, alias file

### DISCUSSION THREAD

**Discussion by kfthor** [VREMSoftwareDevelopment/WiFiAnalyzer#405] [Ideas]
Translation of BSSID mac addresses to an alias string.

#----------WifiAnalyzer alias file----------
#
#Encoding: UTF-8.
#The line starts with # is comment.
#
#Content line format:
#bssid1|alias of bssid1
#bssid2|alias of bssid2
#
18:b1:69:8d:48:78|AP67 8d4876-Sulnasal-BSSID-0
18:b1:69:8d:44:dc|AP66 8d44da 2h-Esja1-BSSID-0

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion

**Comment by ddkgit** [comment]
Don't know if need a file per se, but would love to be able to apply an alias to an entry. I see several devices listed with the same SSID (cos its a mesh setup) and the only way I can tell what's what is looking at the MACs. Sure would be easier if I could assign an alias.
Either way, fantastic app thanks.
